### PR TITLE
Make major and minor changes

### PR DESCRIPTION
**Pull request by sarthak503**

### CODE REVIEW

**Comment by dinskid** [comment]
Is there any reason why the VLA button was removed?

**Comment by dinskid** [comment]
Can you please update the following things:
- Title in students/pg and students/phd should be same as students/undergraduate
- URLs could be made uniform -> /pg to be converted to /postgraduate, /phd to /doctorate
- In /undergraduate, removing IT from "Rules and Regulations for B.Tech.(CSE/ECE/IT)"
- Add a heading in /pg and /phd "Postgraduate (M.Tech.)" "Doctor of Philosophy (PhD)" respectively
- In short, the pages /pg and /phd to be made uniform-looking w.r.t /undergraduate
- Is the word [Download] required additionally to the icon in the pages wherever it is present (about us for instance)? @mayank2021 can you give your views on this?
image

**Comment by mayank2021** [comment]
@dinskid we don't need to tell specifically by writing [Download], as it is the link which predefined to route somewhere.

**Comment by sarthak503** [comment]
Removed the [download] from about us and faculty page

**Comment by dinskid** [comment]
Thanks @sarthak503. Everything seems to be alright!
